docs/oidc_initiation.rst

```diff
@@ -21,14 +21,29 @@ to return an authentication request to the platform
     from lti_tool.views import OIDCLoginInitView
 
     urlpatterns = [
-        path("init/<uuid:registration_uuid/", OIDCLoginInitView.as_view(), name="init"),
-        ...
+        path("init/<uuid:registration_uuid/", OIDCLoginInitView.as_view(
+            main_msg=(
+                "Your browser prevents embedded content from using cookies.  To work "
+                "around this, the content must be opened in a new tab or window.  "
+            ),
+            click_msg="Open a new tab or window now.",
+            loading_msg="Loading..."
+        ), name="init"),
+        # ...
     ]
 
 The ``registration_uuid`` parameter is a reference to the
 :attr:`LtiRegistration.uuid <lti_tool.models.LtiRegistration.uuid>` property, and is
 used to identify the platform registration associated with the initiation request.
 
+The ``main_msg``, ``click_msg``, and ``loading_msg`` **optional** arguments are
+passed to ``DjangoOIDCLogin.enable_check_cookies()`` to generate messages shown
+in the UI.  The values shown here are the default values used by
+``OIDCLoginInitView``.  If cookies are not allowed by the browser, the value of
+``main_msg`` will be shown.  That will be followed by a link with label
+containing the value of ``click_msg``, which opens the content in a new tab or
+window.
+
```

Comment:
I'd add a heading (maybe _Customizing language for blocked cookies_) before this paragraph, and possibly provide a code example after.

Reply:
Added a heading.

My IDE is not rendering ReStructuredText correctly at the moment.  (The IDE was upgraded, but its RST plugin doesn't work with the new IDE.)  If you see any problems with my changes to the documentation formatting, please fix it.  You can edit and commit changes to my branch so they could be neatly added to this PR.